### PR TITLE
#1 updated

### PR DESCRIPTION
**Pull request by birendra2016**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
 # DemoProject
  DemoProject
 
-Modified Changes 1 by John
\ No newline at end of file
+Modified Changes 1 by John
+
+Modified Changes 1 by Birendra
```
Comment (GulabJohn-GPI) [inline README.md]: Why did you add this here?

Comment (birendra2016) [reply README.md]: Simple to test.

Comment (GulabJohn-GPI) [reply README.md]: Okay done, thanks
